### DISCUSSION TITLE
Connecting Prisma with supabase

### DISCUSSION THREAD

**Discussion by Revaycolizer** [supabase/supabase#21149] [Questions] [answered]
# Bug report
## Describe the bug

Trying to migrate prisma schema to my db at Supabase but getting this error Can't reach database server at `db.example.supabase.co`:`5432`
as initially I successfully migrated schema  to some of databases using prisma as I included both DATABASE_URL and DIRECT_URL but using the same approach fails.

## System information

- OS:  Windows
- Version of Node.js: 21.5

**Comment by GaryAustin1** [comment, marked as answer]
Have you moved to Supavisor like has been warned for months now?
https://github.com/orgs/supabase/discussions/17817
And the updated Prisma guide: https://supabase.com/partners/integrations/prisma

You need to change your connection strings.

Moving this to discussion as not a bug.

**Reply by Revaycolizer** [reply]
Seems i was referring to the outdated docs then
